pyuvdata/uvbeam/uvbeam.py

```diff
@@ -1484,8 +1490,8 @@ def _interp_healpix_bilinear(
                 for pol in pols:
                     if pol not in self.polarization_array:
                         raise ValueError(
-                            "Requested polarization {} not found "
-                            "in self.polarization_array".format(pol)
+                            f"Requested polarization {pol} not found "
```

Comment:
f-print statements are the best :D

Reply:
yes! I couldn't resist changing this one.